### PR TITLE
Fixes #20168 - Katello Gpg Key Content-Type

### PR DESCRIPTION
**Pull request by sjha4**

### CODE REVIEW

**Comment by jlsherrill** [comment]
[test]

**Comment by jlsherrill** [comment]
I'm still seeing the reported error, googling around: https://makandracards.com/makandra/25019-when-using-render-text-set-a-content-type  suggests we should be using render(:plain => FOO), maybe give that a try?

File: app/controllers/katello/api/v2/repositories_controller.rb [hunk 1]
```diff
@@ -388,7 +388,7 @@ def import_uploads
     param :id, :identifier, :required => true
     def gpg_key_content
       if @repository.gpg_key && @repository.gpg_key.content.present?
-        render(:text => @repository.gpg_key.content, :layout => false)
+        render(:plain => "#{@repository.gpg_key.content}", :layout => false)
```
Comment (jlsherrill) [inline app/controllers/katello/api/v2/repositories_controller.rb]: the "#{}"  shouldn't actually be needed, `@repository.gpg_key_content` should work just fine

Comment (sjha4) [inline app/controllers/katello/api/v2/repositories_controller.rb]: Thanks @jlsherrill 
Changing the code and refreshing the browser was actually giving a 304 status and not actually re-rendering the new response leading to my confusion about root cause.
Verified that interpolation is unnecessary here. Pushed the requested change. :+1:

**Comment by jlsherrill** [comment]
Thanks @sjha4 !

**Pull request merged**